app/src/main/java/com/infomaniak/mail/ui/main/menuDrawer/MailboxesAdapter.kt

```diff
@@ -127,8 +127,8 @@ class MailboxesAdapter(
 
         itemStyle = if (isInMenuDrawer) SelectionStyle.MENU_DRAWER else SelectionStyle.OTHER
 
-        isPasswordOutdated = !mailbox.isPasswordValid
-        isMailboxLocked = mailbox.isLocked
+        isPasswordOutdated = !mailbox.hasValidPassword
+        isMailboxLocked = mailbox.isLocked || !mailbox.isValidInLDAP
```

Comment:
We could compute a new value isMailboxLocked to be sure that these values are always kept together as you use it twice